### PR TITLE
[Patch-Release] Merged Release CHANGELOG and upgrade patch versions

### PR DESCRIPTION
**Pull request by mssfang**
- Merged CHANGELOG notes of the latest patch version to the main branch
- Upgrade dependency version to the latest patch version

### CODE REVIEW

File: sdk/containerregistry/azure-containers-containerregistry/README.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ and then include the direct dependency in the dependencies section without the v
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-containers-containerregistry</artifactId>
-  <version>1.1.2</version>
+  <version>1.2.0</version>
```
Comment (mssfang) [inline sdk/containerregistry/azure-containers-containerregistry/README.md:59]: surprisingly see this `current` version tag to be updated to a `dependency` tag value. 
But will revert the change anyway.

**Review by vcolin7 — APPROVED** [review]
